governance.md

```diff
@@ -1,133 +1,229 @@
-* Address questions or concerns from the community about project culture, structure, and clarifications of roles, responsibilities and duties
-* Ratify new SIGs
+The steering committee’s responsibility and function lies in the continual shaping of the governance structure to serve the project’s needs best.
 
-Qualifications:
+#### Requirements
 
 * Nominated and Appointed by Steering Committee
   * 2 year terms, term limit of consecutive 4 years (after, individual must take off at least 1 year)
   * Terms should be staggered, which should result in approximately half of the committee up for election each year
   * Term starts w/calendar year (i.e. January 1st)
 
-Logistics
+Election to the SC requires a majority of the SC. If a person is up for reelection they are not allowed to vote for their reelection. 
+
+#### Responsibilities and Privileges
+
+* Set governance structure of the project
+* Address questions or concerns from the community about project culture, structure, and clarifications of roles, responsibilities and duties
+* Ratify new SIGs
+
+#### Logistics
 
 * [Agenda doc](https://docs.google.com/document/d/1HMdwvBPM4uRFqoeAd7eEkVWIC8dQP40zFavOE5Kq-Eg/edit)
 * Email: [sc@spinnaker.io](mailto:sc@spinnaker.io)
 
-Current SC members:
-
-* Andy Glover (Netflix)
-* Matt Duftler (Google)
-* Peter Stout (Netflix)
-* Erin Kidwell (Netflix)
-* Andrew Philips (Google)
+## Documentation Contributors
 
-Election to the SC requires a majority of the SC. If a person is up for reelection they are not allowed to vote for their reelection. 
+The role definitions above were written optimized for code contributors. Documentation contributors will not be held to the same technical requirements. Instead, we look for writers who can express concepts clearly and simply.
 
 ## Special Interest Groups
 
 Special Interest Groups (SIGs) are created as community interest and demand around particular topics become self evident, and continue on as their relevance and needs do. SIGs and their Leads are ratified and green-lit by the Steering Committee via a [SIG proposal process](sig-lifecycle.md). 
 
+## Inactive Members
+
+Members are expected to be continuously active contributors in the community.
+
+A core principle in maintaining a healthy community is encouraging active participation. It is inevitable that peoples' focuses will change over time and they are not expected to be actively contributing forever. However, being a member of the Spinnaker GitHub organization comes with an elevated set of permissions. These capabilities should not be used by those who are unfamiliar with the current state of the Spinnaker project.
+
+Members with an extended period away from the project with no activity will be removed from the Spinnaker GitHub organization and will be required to go through the membership process again after re-familiarizing themselves with the current state. Any time restrictions on a role will be waived when rejoining the organization.
```

Comment:
To make sure I understand...you'd still need to hit the minimum number of PRs (reviewed or authored) when coming back?  Or if they made some contributions and re-familarized themselves would the old ones then still count?

This might not be super important to define right now as it's probably rare, just one thing I wasn't sure of from the phrasing.

Reply:
Clarified that prior contributions will not be counted.